lib/client.dart

```diff
@@ -951,7 +951,12 @@ class Router {
         });
       }
     });
-    route(_history.path);
+    route(_history.path).then((allowed) {
```

Comment:
I was curious about possible errors thrown from calling 'route', but it doesn't look like there is any.